apps/meteor/lib/callbacks.ts

```diff
@@ -42,6 +42,7 @@ type EventLikeCallbackSignatures = {
 	'validateUserRoles': (userData: Partial<IUser>) => void;
 	'workspaceLicenseChanged': (license: string) => void;
 	'afterReadMessages': (rid: IRoom['_id'], params: { uid: IUser['_id']; lastSeen: Date }) => void;
+	'afterReadThread': (message: IMessage, params: { rid: IRoom['_id']; uid: IUser['_id'] }) => void;
```

Comment:
I was wondering if it'd be better to have the messages read in the thread as the first param instead (I mean `messages: IMessage[]`). This callback is used in two places and we have this info ready in one of them
If we do so, there'd be no need to use `findVisibleThreadByThreadId` again in `markThreadMessagesAsRead`
cc @sampaiodiego